package.json

```diff
@@ -102,6 +102,7 @@
     "wait-on": "7.2.0"
   },
   "dependencies": {
+    "@audius/hedgehog": "3.0.0-alpha.1",
```

Comment:
@sliptype does having this dependency addition here make sense as well as it also being in packages/libs?

Reply:
Generally only monorepo tooling should be at the root level, changesets, eslint, that kind of stuff. Is there a reason you need hedgehog at this level?

Reply:
i don't think so? it may be because I npm installed at the root

Reply:
In that case let's remove it!